### ISSUE TITLE
Installation Arybo via pip on Windows, 'ssize_t': redefinition

### ISSUE THREAD

**Issue by trietptm** [quarkslab/arybo#5]
Hi, I install Arybo via pip and Python 3.6.5 32bit on Windows and receive this error:

> C:\Program Files (x86)\python36-32\include\pyconfig.h(174): error C2371: 'ssize_t': redefinition; different basic types
>     include\pa/compat.h(32): note: see declaration of 'ssize_t'

After I comment the lines defining ssize_t in pyconfig.h, I could install Arybo via pip.

**Comment by aguinet** [comment]
Is this still the case with arybo 1.1.0 under Windows/Python 32 bits ?

**Comment by trietptm** [comment]
Yes, it's still the issue with arybo 1.1.0 under Windows Python 3 32 bits because it failed to build pytanque. I could install Arybo with Python 3 64-bit BTW.
```
python37\include\pyconfig.h(167): error C2371: 'ssize_t': redefinition; different basic types
    include\pa/compat.h(32): note: see declaration of 'ssize_t'
```
